[2021-08-12 15:26:34,706] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag_03.processing_tasks.jupyter_task.task_1 2021-08-12T13:26:27.572738+00:00 [queued]>
[2021-08-12 15:26:34,713] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag_03.processing_tasks.jupyter_task.task_1 2021-08-12T13:26:27.572738+00:00 [queued]>
[2021-08-12 15:26:34,713] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 15:26:34,713] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-12 15:26:34,713] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 15:26:34,721] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): processing_tasks.jupyter_task.task_1> on 2021-08-12T13:26:27.572738+00:00
[2021-08-12 15:26:34,725] {standard_task_runner.py:52} INFO - Started process 499445 to run task
[2021-08-12 15:26:34,728] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag_03', 'processing_tasks.jupyter_task.task_1', '2021-08-12T13:26:27.572738+00:00', '--job-id', '126', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task_example.py', '--cfg-path', '/tmp/tmp_fup7vrp', '--error-file', '/tmp/tmp2m7zmmw8']
[2021-08-12 15:26:34,729] {standard_task_runner.py:77} INFO - Job 126: Subtask processing_tasks.jupyter_task.task_1
[2021-08-12 15:26:34,761] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag_03.processing_tasks.jupyter_task.task_1 2021-08-12T13:26:27.572738+00:00 [running]> on host adam
[2021-08-12 15:26:34,790] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag_03
AIRFLOW_CTX_TASK_ID=processing_tasks.jupyter_task.task_1
AIRFLOW_CTX_EXECUTION_DATE=2021-08-12T13:26:27.572738+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-12T13:26:27.572738+00:00
[2021-08-12 15:26:34,791] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-12 15:26:34,791] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo hello']
[2021-08-12 15:26:34,803] {subprocess.py:74} INFO - Output:
[2021-08-12 15:26:34,804] {subprocess.py:78} INFO - hello
[2021-08-12 15:26:34,805] {subprocess.py:82} INFO - Command exited with return code 0
[2021-08-12 15:26:34,834] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag_03, task_id=processing_tasks.jupyter_task.task_1, execution_date=20210812T132627, start_date=20210812T132634, end_date=20210812T132634
[2021-08-12 15:26:34,850] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-12 15:26:34,860] {local_task_job.py:149} INFO - Task exited with return code 0
